.clang-format

```diff
@@ -33,7 +33,7 @@ CompactNamespaces: true
 Cpp11BracedListStyle: true
 FixNamespaceComments: true
 IndentWidth: 2
-IncludeBlocks: Regroup
+IncludeBlocks: Preserve
```

Comment:
Why do we need to change this? This leads to changes in EVERY .hpp and .cpp file.

Reply:
All the existing files will get formatted just as it was.

The only difference is this - when you have a groups of includes separated by empty lines, previously - it would merge all them into one group without empty lines, and sort them all alphabetically. Now - it would sort each group individually and won't merge them.

So, now I can have headers formatted like this:
```
#include <http/policy.hpp>

#include <algorithm>
#include <cstdlib>
#include <limits>
#include <thread>
```

Previously, I couldn't. It would insert `http` in the middle alphabetically. Note that it still does sort! And if you chose to have no groups, i.e. not separate groups of includes by spaces, it works exactly as before. Only that before it would force you to have no groups.

Reply:
i'm okay with this.